browser-test/jest.config.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
-  preset: 'ts-jest',
-  testMatch: ['**/__tests__/**/*.+(ts|js)', '**/?(*.)+(spec|test).+(ts|js)'],
```

Comment:
do you know what the __tests__ match was matching?  EG what is the consequence of removing it?

Reply:
Looks like default jest convention for organizing tests. https://github.com/facebook/jest/issues/637 
We are not using it.